ethereum/rpc/namespaces/eth/api.go

```diff
@@ -985,7 +983,8 @@ func (e *PublicAPI) GetTransactionReceipt(hash common.Hash) (map[string]interfac
 // and have a from address that is one of the accounts this node manages.
 func (e *PublicAPI) PendingTransactions() ([]*rpctypes.RPCTransaction, error) {
 	e.logger.Debugln("eth_getPendingTransactions")
-	return e.backend.PendingTransactions()
+
+	return []*types.RPCTransaction{}, nil
```

Comment:
why empty?

Reply:
To implement `eth_getPendingTransactions`, we have to do more queries to fetch block information, maybe do it in another issue.

Reply:
https://github.com/tharsis/ethermint/issues/244